frontend/src/v4/routes/components/textField/textField.styles.ts

```diff
@@ -71,9 +71,7 @@ export const StyledMarkdownField = styled(MarkdownField)<{ $isPlaceholder?: bool
 		font-size: 14px;
 		overflow: hidden;
 
-		/* TODO: fix after new palette is released */
-		${({ $isPlaceholder }) => $isPlaceholder && css`color: #C1C8D5;`}
-
+		${({ $isPlaceholder }) => $isPlaceholder && css`color: ${({ theme }) => theme.palette.base.lighter};`}
```

Comment:
Same as previous comment, check if this works in v4

Reply:
I have tested and they both work